v2/internal/util/metrictree/namespace.go

```diff
@@ -194,7 +194,7 @@ func newStaticAnyElement() *staticAnyElement {
 }
 
 func (*staticAnyElement) Match(s string) bool {
-	return isValidIdentifier(s)
+	return s != "*" && s != "**" && isValidGroupIdentifier(s)
```

Comment:
I'm ok with this change but let's wait for @adamiklukasz to confirm if it won't have some side effects in other places